cal-adapt/climakitae
# Add back in derived variables, reorder dropdown

In the original catalog restructure PR, I realized I forgot to add in the functionality to derive the hourly variables: precipitation total, wind magnitude, and relative humidity. I add that back in here, branched off of Eric's branch `code_cleanup` (which in turn is branched off of `catalog_restructure`.) This fix required me to change up data_loaders.py a bit. 

I also reordered the variables in variable_catalog.csv and cleaned up the descriptions for hourly and daily/monthly qv, along with snowfall. 

**To test:** 
Try selecting and loading the derived hourly variables and make sure the data looks correct, and the unit conversion options for those variables look good. Confirm that you agree with the ordering for the daily/monthly variables.
